src/cbc_sdk/platform/vulnerability_assessment.py

```diff
@@ -211,6 +257,21 @@ def set_vcenter(self, vcenter_uuid):
             self._vcenter_uuid = vcenter_uuid
         return self
 
+    def set_visibility(self, visibility):
+        """
+        Restricts the vulnerabilities that this query is performed on to the specified visibility
+
+        Args:
+            visibility (str): The visibility state of the vulnerabilty. (supports ACTIVE, ALL, DISMISSED)
+
+        Returns:
+            VulnerabilityQuery: This instance.
```

Comment:
Maybe this should be returning VulnerabilityOrgSummaryQuery.

Reply:
Yes, it should.